fdd-publish-5.0/fdd-publish.csproj

```diff
@@ -0,0 +1,6 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
`netcoreapp5.0` -> `net5.0`

Reply:
Part of the test runner will automagically update the target framework in this file.